### ISSUE TITLE
Are we indexing the right text?

### ISSUE THREAD

**Issue by mccalluc** [WGBH/AAPB2#58]
@sroosa : Right now, I'm indexing the text nodes in the xml, ie, 
```xml
<not-this nor-this="or this">but this does get indexed</not-this>
```

- Should the attributes be indexed too?
- Or do we even want to index all the text? There are a lot of details, for instance in the instantiation, that don't show up on the screen: Maybe we should only see the things that can be seen without adding a special .pbcore suffix to the url
- Or maybe we add one more constrained search: The default searches the visible text, but there is also an option to match anything (even element names?) in the pbcore.

**Comment by sroosa** [comment]
Hmm, I think that I'd feel better indexing more rather than less for search. I'd rather not constrain to only what appears on the screen. 
The only attribute I can think that would be useful to search, however, is creator/contributor affiliation. @caseyedavis12 might want to weigh in on this.

**Comment by mccalluc** [comment]
ok, for now I'm going to tweak it to just search the text we actually show.

**Comment by mccalluc** [comment]
fixed by #103
